src/features/CourtCaseList/CourtCaseListEntry.tsx

```diff
@@ -107,6 +115,7 @@ const CourtCaseListEntry: React.FC<Props> = ({ courtCase, currentUser }: Props)
           ) : (
             <LockedByTag lockedBy={triggerLockedByUsername} />
           )}
+          {<CaseUnlockedTag isCaseUnlocked={!exceptionHasBeenRecentlyUnlocked && triggerHasBeenRecentlyUnlocked} />}
```

Comment:
Is it necessary to check !exceptionHasBeenRecentlyUnlocked since there is a separate lock for triggers/errors

Reply:
It's a result of some experimenting, in same cases I had the badge on 2 things.